modules/claims_api/app/controllers/claims_api/v0/forms/power_of_attorney_controller.rb

```diff
@@ -100,6 +102,14 @@ def schema
 
         private
 
+        def current_poa_code
+          current_poa.try(:code)
+        end
+
+        def current_poa
+          @current_poa ||= BGS::PowerOfAttorneyVerifier.new(target_veteran).current_poa
+        end
```

Comment:
i dont think these are needed if
https://github.com/department-of-veterans-affairs/vets-api/pull/6746/files#r625204314
is addressed